### PR TITLE
Introduce a Cloudflare health worker

### PR DESCRIPTION
**Pull request by si14**
This PR introduces a new Cloudflare worker for health checks.

At the moment the worker only translates Updown webhooks into Discord webhooks. In the future we can teach this worker to check more things.

### Change Type

- [x] `internal` — Any other changes that don't affect the published package

### CODE REVIEW

File: scripts/deploy.ts [hunk 1]
```diff
@@ -100,16 +103,19 @@ async function main() {
...
-	await discordStep('[4/6] deploying asset uploader to cloudflare', async () => {
+	await discordStep('[4/7] deploying asset uploader to cloudflare', async () => {
 		await deployAssetUploadWorker({ dryRun: false })
 	})
-	await discordStep('[5/6] deploying multiplayer worker to cloudflare', async () => {
+	await discordStep('[5/7] deploying multiplayer worker to cloudflare', async () => {
 		await deployTlsyncWorker({ dryRun: false })
 	})
+	await discordStep('[6/7] deploying health worker to cloudflare', async () => {
+		await deployHealthWorker({ dryRun: false })
```
Comment (ds300) [inline scripts/deploy.ts:113]: you should add a `dryRun: true` version up above in the 'cloudflare deploy dry run' zone

File: apps/health-worker/src/discord.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import { exhaustiveSwitchError } from '@tldraw/utils'
+import { type APIEmbed } from 'discord-api-types/v10'
+import { Event as UpdownEvent } from './updown_types'
+
+// discord wants decimal colours
+const GREEN = 4243543
+const RED = 14692657
+const ORANGE = 16213767
```
Comment (ds300) [inline apps/health-worker/src/discord.ts:8]: weird, never seen colors passed around like this.

**Review by ds300 — APPROVED** [review]
Nice one! 🙏🏼

File: apps/health-worker/src/discord.ts (hunk 2 above)
Comment (si14) [reply apps/health-worker/src/discord.ts:8]: yeah, it's a weird API

File: scripts/deploy.ts (hunk 1 above)
Comment (si14) [reply scripts/deploy.ts:114]: fixed

**Pull request merged**